### PR TITLE
[Docs][Connector-V2][MyHours]Reconstruct the MyHours connector document

### PR DESCRIPTION
**Pull request by TaoZex**
## Purpose of this pull request

subtask of https://github.com/apache/seatunnel/issues/4572

Reconstruct the MyHours connector document

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

File: docs/en/connector-v2/source/MyHours.md [hunk 1]
```diff
@@ -15,71 +17,104 @@ Used to read data from My Hours.
...
+- [x] [batch](../../concept/connector-v2-features.md)
+- [ ] [stream](../../concept/connector-v2-features.md)
+- [ ] [exactly-once](../../concept/connector-v2-features.md)
+- [ ] [column projection](../../concept/connector-v2-features.md)
+- [ ] [parallelism](../../concept/connector-v2-features.md)
+- [ ] [support user-defined split](../../concept/connector-v2-features.md)
 
-### retry_backoff_multiplier_ms [int]
+Supported DataSource Info
+-------------------------
```
Comment (Hisoka-X) [inline docs/en/connector-v2/source/MyHours.md]:
```suggestion
## Supported DataSource Info
```

Comment (TaoZex) [reply docs/en/connector-v2/source/MyHours.md]: Thanks for your reminding, I have modified it.

**Comment by zhilinli123** [comment]
Add a connector version for this : https://github.com/apache/seatunnel/issues/4572 @TaoZex

File: docs/en/connector-v2/source/MyHours.md [hunk 2]
```diff
@@ -15,71 +17,103 @@ Used to read data from My Hours.
...
+         id = string
+       }
+    }
+}
 
-The maximum retry-backoff times(millis) if request http failed
+# Console printing of the read data
+sink {
+  Console {
+    parallelism = 1
```
Comment (hailin0) [inline docs/en/connector-v2/source/MyHours.md:109]:
```suggestion
```

File: docs/en/connector-v2/source/MyHours.md [hunk 3]
```diff
@@ -131,13 +165,7 @@ connector will generate data as the following:
...
 | {"code":  200, "data":  "get success", "success":  true} |
 
-### schema [Config]
-
-#### fields [Config]
-
-the schema fields of upstream data
-
-### content_json [String]
+### content_json
```
Comment (EricJoy2048) [inline docs/en/connector-v2/source/MyHours.md:168]: Can you use real data obtained from MyHours as an example to illustrate？

File: docs/en/connector-v2/source/MyHours.md [hunk 4]
```diff
@@ -212,14 +240,14 @@ Here is an example:
 - Test data can be found at this link [mockserver-config.json](../../../../seatunnel-e2e/seatunnel-connector-v2-e2e/connector-http-e2e/src/test/resources/mockserver-config.json)
 - See this link for task configuration [http_contentjson_to_assert.conf](../../../../seatunnel-e2e/seatunnel-connector-v2-e2e/connector-http-e2e/src/test/resources/http_contentjson_to_assert.conf).
 
-### json_field [Config]
+### json_field
```
Comment (EricJoy2048) [inline docs/en/connector-v2/source/MyHours.md:243]: Same as above.

**Comment by TaoZex** [comment]
> Add a connector version for this : #4572 @TaoZex

myhours is an http connector that does not require a specific version

**Comment by zhilinli123** [comment]
> > Add a connector version for this : #4572 @TaoZex
> 
> myhours is an http connector that does not require a specific version

Done

**Review by ic4y — APPROVED** [review]

**Review by liugddx — APPROVED** [review]
+1

**Pull request merged**